### PR TITLE
OY2-26604 - FAQ for Withdraw RAI Response

### PR DESCRIPTION
**Pull request by bflynn-cms**
Story: https://qmacbis.atlassian.net/browse/OY2-26604
Endpoint: See github-actions bot comment

### Details

Added FAQ question and answers for withdraw RAI response of Medicaid SPA, CHIP SPA, and Waiver.

### Changes

- Added to FAQContent.tsx

### Implementation Notes

- Structure isnt clear from story and it looked odd to have it all bulletized so I winged it a bit

### Test Plan

1. Navigate to FAQ page
2. Validate new Q&As

### CODE REVIEW

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 1]
```diff
@@ -555,6 +555,62 @@ export const oneMACFAQContent: FAQContent[] = [
...
+              </li>
+            </ul>
+            <p>
+              After receiving confirmation from your CMS Point of Contact that
+              the Withdraw Formal RAI Response feature has been enabled, locate
+              and select the Medicaid SPA submission package.
+            </p>
+            <p>
+              The package status remains as Under Review and a substatus of RAI
+              Response Withdraw Enabled will be reflected below the status for
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:586]: the substatus is: Withdraw Formal RAI Response Enabled  -- I know, the AC of the story is wrong...

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 2]
```diff
@@ -641,6 +697,62 @@ export const oneMACFAQContent: FAQContent[] = [
...
+              </li>
+            </ul>
+            <p>
+              After receiving confirmation from your CMS Point of Contact that
+              the Withdraw Formal RAI Response feature has been enabled, locate
+              and select the CHIP SPA submission package.
+            </p>
+            <p>
+              The package status remains as Under Review and a substatus of RAI
+              Response Withdraw Enabled will be reflected below the status for
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:728]: same here: Withdraw Formal RAI Response Enabled

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 3]
```diff
@@ -1152,6 +1264,63 @@ export const oneMACFAQContent: FAQContent[] = [
...
+              </li>
+            </ul>
+            <p>
+              After receiving confirmation from your CMS Point of Contact that
+              the Withdraw Formal RAI Response feature has been enabled, locate
+              and select the Medicaid Waiver submission package.
+            </p>
+            <p>
+              The package status remains as Under Review and a substatus of RAI
+              Response Withdraw Enabled will be reflected below the status for
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:1296]: unsurprisingly... Withdraw Formal RAI Response Enabled

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 4]
```diff
@@ -555,6 +555,62 @@ export const oneMACFAQContent: FAQContent[] = [
...
+            </p>
+            <p>
+              <b>
+                Note: These submissions will remain on the clock until the
+                package action has been taken.
+              </b>
+            </p>
+            <ul>
+              <li>
+                On the Formal RAI Response Withdraw Details page, upload any
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:597]: Details page?  I think this should be form page... story AC again

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 5]
```diff
@@ -641,6 +697,62 @@ export const oneMACFAQContent: FAQContent[] = [
...
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:739]: Details page?  I think this should be form page... story AC again

File: services/ui-src/src/libs/faq/faqContent.tsx [hunk 6]
```diff
@@ -1152,6 +1264,63 @@ export const oneMACFAQContent: FAQContent[] = [
...
```
Comment (kristin-at-theta) [inline services/ui-src/src/libs/faq/faqContent.tsx:1307]: Details page?  I think this should be form page... story AC again

**Review by kristin-at-theta — APPROVED** [review]
Always weird when a story matches the AC and yet... we know there will be changes...

The subStatus should read "Withdraw Formal RAI Response Enabled" and the page to submit is the form page... though, now that I look at the form page, the top section is labelled: "Withdraw Formal RAI Response Details" so that's probably where that came from ...

Approved because AC is matched, but I won't be surprised if this changes...

**Comment by bflynn-cms** [comment]
> Always weird when a story matches the AC and yet... we know there will be changes...
> 
> The subStatus should read "Withdraw Formal RAI Response Enabled" and the page to submit is the form page... though, now that I look at the form page, the top section is labelled: "Withdraw Formal RAI Response Details" so that's probably where that came from ...
> 
> Approved because AC is matched, but I won't be surprised if this changes...

Discussed with Kristin and decided it would be better to cut it off at the pass so to speak and just incorporate the changes she suggested rather than follow word for word from ACs.

**Pull request merged**